Rakefile

```
@@ -37,3 +37,17 @@ EOS
   sh "git add version"
   sh "git commit -m 'Bump version'"
 end
+
+begin
+  require 'rubygems'
+  require 'github_changelog_generator/task'
+
+  GitHubChangelogGenerator::RakeTask.new :changelog do |config|
+    config.exclude_labels = %w{duplicate question invalid wontfix wont-fix skip-changelog}
+    config.user = 'voxpupuli'
+    config.project = 'puppet-blacksmith'
+    gem_version = Gem::Specification.load("#{config.project}.gemspec").version
+    config.future_release = gem_version
+  end
+rescue LoadError
+end
```

I've started to like this pattern which is possible since Ruby 2.5.
```suggestion
begin
  require 'rubygems'
  require 'github_changelog_generator/task'
rescue LoadError
else
  GitHubChangelogGenerator::RakeTask.new :changelog do |config|
    config.exclude_labels = %w{duplicate question invalid wontfix wont-fix skip-changelog}
    config.user = 'voxpupuli'
    config.project = 'puppet-blacksmith'
    gem_version = Gem::Specification.load("#{config.project}.gemspec").version
    config.future_release = gem_version
  end
end
```